### PR TITLE
Feature/authorization admin

### PR DESCRIPTION
**Pull request by WalterGoldenOwl**
Description Pull Request:

## Todo
- Implement UI and logic Review Business Accounts Screen
## Task
Link of task: 

1. https://trello.com/c/9cnQ8Nqt
2. https://trello.com/c/pd3mSUoT

## Screenshot
Screenshot_1698632038
Screenshot_1698632463

## Video

https://github.com/dustin-goldenowl/jobspot/assets/144195318/7bba6285-42c9-41c6-a4a0-5996b71d6bc4

### CODE REVIEW

File: lib/src/core/utils/prefs_utils.dart [hunk 1]
```diff
@@ -10,6 +10,30 @@ class PrefsUtils {
...
+
+  static String? get email => prefs.getString("email");
+
+  static String? get password => prefs.getString("password");
+
+  static Future setRemember({
+    required String email,
+    required String password,
+  }) async {
+    await setIsRemember(true);
```
Comment (dustin-goldenowl) [inline lib/src/core/utils/prefs_utils.dart]: replace to await Future.wait([ request ])

File: lib/src/core/utils/prefs_utils.dart [hunk 2]
```diff
@@ -10,6 +10,30 @@ class PrefsUtils {
...
+    await prefs.setString("email", email);
+    await prefs.setString("password", password);
+  }
+
+  static Future removeRemember() async {
+    await setIsRemember(false);
```
Comment (dustin-goldenowl) [inline lib/src/core/utils/prefs_utils.dart]: the same above

File: lib/src/core/utils/prefs_utils.dart (hunk 1 above)
Comment (WalterGoldenOwl) [reply lib/src/core/utils/prefs_utils.dart]: I fixed it

File: lib/src/core/utils/prefs_utils.dart (hunk 2 above)
Comment (WalterGoldenOwl) [reply lib/src/core/utils/prefs_utils.dart]: I fixed it

**Pull request merged**